### ISSUE TITLE
Support for Python 3.9

### ISSUE THREAD

**Issue by UndeadKernel** [biolab/orange3#5154]
- [ ] **What's wrong?**
It seems that installing Orange3 is not possible when using Python 3.9. In my Arch GNU/Linux system, a command such as `pip install orange3` fails, with the relevant errors being:
```
Orange/classification/_tree_scorers.c:24988:25: error: 'PyTypeObject' {aka 'struct _typeobject'} has no member named 'tp_print'
```
In Python 3.9, the function `tp_print` has been removed (see https://bugs.python.org/issue39361).

Are there plans to support Python 3.9?

**Comment by janezd** [comment]
Thanks for reporting this.

Which version of Cython do you have? I just tried with the latest version from conda-forge and it works.

**Comment by markotoplak** [comment]
I think this was solved in master with #5080, but that fix is not part of an official release yet. Does installing from the master branch work? If you are using pip, you can try `pip install https://github.com/biolab/orange3/archive/master.zip`

**Comment by UndeadKernel** [comment]
Thanks for the help @janezd and @marinkaz. I can confirm that installing from the master branch works as expected using Python 3.9 in Arch.
